framework/src/Volo.Abp.BlazoriseUI/BlazoriseUiNotificationService.cs

```diff
@@ -7,34 +10,79 @@
 namespace Volo.Abp.BlazoriseUI
 {
     [Dependency(ReplaceServices = true)]
-    public class BlazoriseUiNotificationService : IUiNotificationService, ITransientDependency
+    public class BlazoriseUiNotificationService : IUiNotificationService, IScopedDependency
     {
+        /// <summary>
+        /// An event raised after the notification is received.
+        /// </summary>
+        public event EventHandler<UiNotificationEventArgs> NotificationReceived;
+
+        private readonly IStringLocalizer<AbpUiResource> localizer;
+
         public ILogger<BlazoriseUiNotificationService> Logger { get; set; }
 
-        public BlazoriseUiNotificationService()
+        public BlazoriseUiNotificationService(
+            IStringLocalizer<AbpUiResource> localizer)
         {
+            this.localizer = localizer;
+
             Logger = NullLogger<BlazoriseUiNotificationService>.Instance;
         }
 
-        public Task Info(string message)
+        public Task Info(string message, string title = null, Action<UiNotificationOptions> options = null)
         {
+            var uiNotificationOptions = CreateDefaultOptions();
+            options?.Invoke(uiNotificationOptions);
+
             Logger.LogInformation(message);
```

Comment:
No need to log the notifications. I suppose it was a test code and you didn't delete it.

Reply:
Logging removed